Makefile

```diff
@@ -1,13 +1,17 @@
-.PHONY: lint test mock-gen
-
-MOCKS_DIR=./testutil/mocks
+MOCKS_DIR := ./testutil/mocks
 
 OPFGD_PKG := github.com/babylonlabs-io/finality-gadget/cmd/opfgd
 
+BUILDDIR ?= $(CURDIR)/build
+BUILD_FLAGS := --tags '$(BUILD_TAGS)' --ldflags '$(LDFLAGS)'
+BUILD_ARGS := $(BUILD_ARGS) -o $(BUILDDIR)
+
+DOCKER ?= $(shell which docker)
+GIT_ROOT := $(shell git rev-parse --show-toplevel)
+
 mock-gen:
 	go install go.uber.org/mock/mockgen@latest
-	mockgen -source=sdk/client/expected_clients.go -package mocks -destination $(MOCKS_DIR)/expected_clients_mock.go
-	mockgen -source=sdk/client/interface.go -package mocks -destination $(MOCKS_DIR)/sdkclient_mock.go
+	mockgen -source=finalitygadget/expected_clients.go -package mocks -destination $(MOCKS_DIR)/expected_clients_mock.go
```

Comment:
good catch. can you also run mock-gen to see if anything changes?

Reply:
I had run `make mock-gen`, no change.